openstack_controller/tests/test_unit_ironic.py

```diff
@@ -539,3 +539,117 @@ def test_conductors_metrics(aggregator, check, dd_run_check, metrics):
             value=metric['value'],
             tags=metric['tags'],
         )
+
+
+@pytest.mark.parametrize(
+    ('connection_baremetal', 'paginated_limit', 'instance', 'metrics', 'api_type', 'expected_api_call_count'),
+    [
+        pytest.param(
+            None,
+            1,
+            configs.REST,
+            CONDUCTORS_METRICS_IRONIC_MICROVERSION_DEFAULT,
+            ApiType.REST,
+            2,
+            id='api rest no microversion low limit',
+        ),
+        pytest.param(
+            None,
+            1000,
+            configs.REST,
+            CONDUCTORS_METRICS_IRONIC_MICROVERSION_DEFAULT,
+            ApiType.REST,
+            2,
+            id='api rest no microversion high limit',
+        ),
+        pytest.param(
+            None,
+            1,
+            configs.REST_IRONIC_MICROVERSION_1_80,
+            CONDUCTORS_METRICS_IRONIC_MICROVERSION_1_80,
+            ApiType.REST,
+            1,
```

Comment:
```suggestion
            2,
```
I think this should be 2 since it requests 2 conductors

Reply:
Agreed, I've fixed it.